app/presenters/tree_builder_ops_rbac.rb

```diff
@@ -38,6 +40,7 @@ def x_get_tree_custom_kids(object_hash, options)
   end
 
   def x_get_tree_tenant_kids(object, options)
+    options.merge!(:expand => false)
```

Comment:
Kind of strange that we update the incoming options hash here and then not use it. Unexpected side effect for the caller.